### DISCUSSION TITLE
grouping all the nodes into one cluster

### DISCUSSION THREAD

**Discussion by WanyongFeng** [mapequation/infomap#241] [Q&A] [answered]
Hi, I was using the infomap on an undirected, weighted graph, with 121 nodes and 6225 edges. The issue I have is that all the elements were grouped into one cluster. Can someone help me with that, thank you!

**Comment by antoneri** [comment, marked as answer]
Hi, and thanks for reaching out!

It sounds like your network Is very dense, making it hard to find a meaningful community structure.

The usual way to handle these situations is to use the flag `--markov-time f` for some `f` smaller than 1.

You can also consider thresholding how many links you include to make the network more sparse with `--weight-threshold n`.

If the data is aggregated and you can stratify it into multiple layers, that would also help.

Let me know how it goes!
